File: day_1/day_1.Python/program.py
import builtins
from day_1_Core.program import (distance, product_of_input)
from fable_modules.fable_python.stdlib.builtins_ import print

file1: str = builtins.open("..\\input1.txt", mode = "r").read()

file2: str = builtins.open("..\\input2.txt", mode = "r").read()

print(distance(file1))

print(distance(file2))

print(product_of_input(file1))

print(product_of_input(file2))

File: day_1/day_1.Python/Program.fs
open Fable.Python.Builtins
open Day1Core

let file1 = builtins.``open``("..\input1.txt", "r").read()
let file2 = builtins.``open``("..\input2.txt", "r").read()

print(distance file1)
print(distance file2)
print(productOfInput file1)
print(productOfInput file2)
